Added docs for case insensitive feature in dq (#43)
CiscoTestAutomation/genie: docs/userguide/utils/index.rst

  >>> output.q.contains('[1,2]', regex=True)
  >>> output.q.contains('.*ware', regex=True)

> To do a case insensitive comparison in contains you have to set regex and ignore_case variables to ``True``.

Reviewer: ```suggestion
To do a case insensitive comparison in contains you have to set both regex and ignore_case variables to ``True``.
```

Author: updated

  # If searching for both key and value using regex
  >>> output.q.contains_key_value('slot/world_wide_name|mac.*|model', '[a-zA-Z0-9\-\s]+', key_regex=True, value_regex=True)

> Similar to contains here also you can do case insensitive comparison.

Reviewer: ```suggestion
Similar to ``contains`` here also you can do case insensitive comparison.
```

Author: updated

> - If the key have to be case insensitive then you have to set ``key_regex`` and ``ignore_case_key`` as True.

Reviewer: ```suggestion
- If the ``key`` has to be case insensitive then you have to set both ``key_regex`` and ``ignore_case_key`` as True.
```

Author: updated

> - If the value have to be case insensitive then you have to set ``value_regex`` and ``ignore_case_value`` as True.

Reviewer: ```suggestion
- If the ``value`` has to be case insensitive then you have to set both ``value_regex`` and ``ignore_case_value`` as True.
```

Author: updated

  # if applying regex only for value set value_regex=True
  >>> output.q.not_contains_key_value('lc', '(3|4)', value_regex=True)

> This one also supports case insensitive which was already explained for contains_key_value api.

Reviewer: ```suggestion
This one also supports both ``ignore_case_key`` and ``ignore_case_value`` which was already explained for contains_key_value api.
```

Author: updated
